Do not use -Wno-psabi option

This option was added in gcc 4.5, and Buildroot still uses gcc 4.3 on
some architectures. Since it's non essential (only disables some
warning), let's get rid of it.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: b/configure.ac
===================================================================
--- a/configure.ac
+++ b/configure.ac
@@ -137,7 +137,7 @@
 use_udev="no"
 use_adapter_detection="yes"
 use_lockdev="no"
-SUPPRESS_MANGLING_WARNINGS=" -Wno-psabi"
+SUPPRESS_MANGLING_WARNINGS=""
 case "${host}" in
   *-*-linux*)
     ## search for udev if pkg-config was found
